### ISSUE TITLE
FR: Default Pricing Plan

### ISSUE THREAD

**Issue by Makisuo** [tierrun/tier#291]
### What are you trying to do?

Be able to set one pricing plan as the default, which is automatically set for every newly registered account.

### How should we solve this?

A new attribute for pricing.json, declaring a plan as default.

### What is the impact of not solving this?

It can be worked around by manually assigning the plan after registering a new user, though I feel like this would be quite an essential feature since most SaaS will have a free tier that always will need to be assigned. 
So this would affect most tier users in a positive way if implemented.

**Comment by isaacs** [comment]
What api would you imagine this looking like? You'll have to hit tier at *some* point, if only to create the org. So you'd need some kind of "subscribe ${org} to ${the default plan} if they don't already have a subscription", right?

If we had a "subscribe if no subscription" option, you could just set the default plan in an env or something, at least, and just always call it on login/signup.

**Comment by Makisuo** [comment]
Yes thats excactly what I meant, basically giving a way to subscribe to default on singup if not subscribed, automatically.

With an env would work, though I feel like this is a very common use case so it might be useful to implement it in the PricingSchema itself and the logic be handled in the tier itself.

As far as I see it, in most cases there will always be a Free tier to track usage so most would benefit from this better DX imo.
